src/main/kotlin/com/ampnet/reportserviceeth/service/data/TransactionFactory.kt

```diff
@@ -68,20 +71,26 @@ abstract class Transaction(transaction: TransactionInfo) {
     }
 }
 
-class TransactionInvest(transaction: TransactionInfo) : Transaction(transaction) {
-    override val txStatus = TransactionStatusType.PAID_OUT
+class TransactionReserveInvestment(transaction: TransactionInfo) : Transaction(transaction) {
+    override val txStatus = TransactionStatusType.PAID_IN
     override val name: String
-        get() = translations.investment
+        get() = translations.reserveInvestment
 }
 
 class TransactionCancelInvestment(transaction: TransactionInfo) : Transaction(transaction) {
-    override val txStatus = TransactionStatusType.PAID_IN
+    override val txStatus = TransactionStatusType.PAID_OUT
     override val name: String
-        get() = translations.investmentCancel
+        get() = translations.cancelInvestment
 }
 
 class TransactionSharePayout(transaction: TransactionInfo) : Transaction(transaction) {
-    override val txStatus = TransactionStatusType.PAID_IN
+    override val txStatus = TransactionStatusType.PAID_OUT
     override val name: String
```

Comment:
If you change the logic for in/out transactions, add the explanation in the TransactionStatusType (javadoc).